src/proto/h2/server.rs

```diff
@@ -35,7 +35,7 @@ const DEFAULT_CONN_WINDOW: u32 = 1024 * 1024; // 1mb
 const DEFAULT_STREAM_WINDOW: u32 = 1024 * 1024; // 1mb
 const DEFAULT_MAX_FRAME_SIZE: u32 = 1024 * 16; // 16kb
 const DEFAULT_MAX_SEND_BUF_SIZE: usize = 1024 * 400; // 400kb
-// 16 MB "sane default" taken from golang http2
+                                                     // 16 MB "sane default" taken from golang http2
```

Comment:
```suggestion
// 16 MB "sane default" taken from golang http2
```

Reply:
Rustfmt was doing this, but ill take this for the same of a small diff